static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -185,6 +186,21 @@ define([
         return false;
     };
 
+    Id.refreshCookie = function () {
+        if (Id.isUserLoggedIn()) {
+            var lastRefresh = storage.local.get(Id.lastRefreshKey),
```

Comment:
Yes see 'should return true for a user who has never refreshed cookies' in Api.spec